cypress/page/plp.page.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="Cypress" />
+class PLPPage {
+
+    public addToCartButton(): void {
+        cy.get(" div.button-container > a.button.ajax_add_to_cart_button").click();
+    }
+
+    public proceedToCheckoutButton(): void {
+        cy.get("a[title='Proceed to checkout']").click();
+    }
+}
+
+export { PLPPage }
```

Comment:
EOF

Why this name, I think this isn't one of the names given in the workshop

Reply:
PLP = product listing page, and in the workshop there is no name suggested actually